<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>coption</class>
 <widget class="QDialog" name="coption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="opSave">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="opClose">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>530</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="optionTab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>721</width>
     <height>511</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Datebase</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>701</width>
       <height>461</height>
      </rect>
     </property>
     <property name="title">
      <string>Datebase strting</string>
     </property>
     <widget class="QRadioButton" name="radioMysql">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;mysql database&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>mysql</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioSqlite">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;sqlite datebase not set config&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>sqlite</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>use database:</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>140</y>
        <width>201</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="mysqlHost"/>
       </item>
       <item>
        <widget class="QLineEdit" name="mysqlUser"/>
       </item>
       <item>
        <widget class="QLineEdit" name="mysqlPasw"/>
       </item>
       <item>
        <widget class="QLineEdit" name="mysqlDb"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>81</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Dbhost:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Dbuser:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Datebase:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>radioMysql</zorder>
     <zorder>radioSqlite</zorder>
     <zorder>label_5</zorder>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Url setting</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>701</width>
       <height>461</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>699</width>
        <height>459</height>
       </rect>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>691</width>
         <height>451</height>
        </rect>
       </property>
       <property name="title">
        <string>web class</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>50</y>
          <width>291</width>
          <height>311</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="tag1"/>
         </item>
         <item>
          <widget class="QLineEdit" name="tag2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="tag3"/>
         </item>
         <item>
          <widget class="QLineEdit" name="tag4"/>
         </item>
         <item>
          <widget class="QLineEdit" name="tag5"/>
         </item>
         <item>
          <widget class="QLineEdit" name="tag6"/>
         </item>
         <item>
          <widget class="QLineEdit" name="tag7"/>
         </item>
         <item>
          <widget class="QLineEdit" name="tag8"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>41</width>
          <height>311</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>tag1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>tag2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>tag3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>tag4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>tag5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>tag6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>tag7</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>tag8</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>20</y>
          <width>81</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>start lable</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>50</y>
          <width>291</width>
          <height>311</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLineEdit" name="tage1"/>
         </item>
         <item>
          <widget class="QLineEdit" name="tage2"/>
         </item>
         <item>
          <widget class="QLineEdit" name="tage3"/>
         </item>
         <item>
          <widget class="QLineEdit" name="tage4"/>
         </item>
         <item>
          <widget class="QLineEdit" name="tage5"/>
         </item>
         <item>
          <widget class="QLineEdit" name="tage6"/>
         </item>
         <item>
          <widget class="QLineEdit" name="tage7"/>
         </item>
         <item>
          <widget class="QLineEdit" name="tage8"/>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>20</y>
          <width>64</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>end lable</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>tables setting</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>701</width>
       <height>461</height>
      </rect>
     </property>
     <property name="title">
      <string>sql table setting</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>110</y>
        <width>231</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLineEdit" name="column1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="column2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="column3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="column4"/>
       </item>
       <item>
        <widget class="QLineEdit" name="column5"/>
       </item>
       <item>
        <widget class="QLineEdit" name="column6"/>
       </item>
       <item>
        <widget class="QLineEdit" name="column7"/>
       </item>
       <item>
        <widget class="QLineEdit" name="column8"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>191</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLineEdit" name="old_class1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="old_class2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="old_class3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="old_class4"/>
       </item>
       <item>
        <widget class="QLineEdit" name="old_class5"/>
       </item>
       <item>
        <widget class="QLineEdit" name="old_class6"/>
       </item>
       <item>
        <widget class="QLineEdit" name="old_class7"/>
       </item>
       <item>
        <widget class="QLineEdit" name="old_class8"/>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="table_name">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>60</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>table name:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Robot</string>
    </attribute>
    <widget class="QCheckBox" name="is_emdata">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>301</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Collection empty data</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="is_must">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>301</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>must contain data</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="musttext">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="musttext_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>107</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>||</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sleep_time">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>150</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>sleep time</string>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>optionTab</zorder>
  <zorder>opSave</zorder>
  <zorder>opClose</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
